### PR TITLE
Added collection for lodgement form + fixes

### PR DESCRIPTION
**Pull request by DavidCrosswellGSQ**
Added a collection to display on the lodgement portal form selector. 

SRA need a collection very soon. However, we can add to this collection over the next days.

The objective of this collection is to present industry with an  input control on the lodgement portal form to select when they are submitting reports and data to GSQ.

It needs to be something that meets the needs of both industry and GSQ.

The current ESDC https://vocabs.gsq.digital/vocabulary/earth-science-data-category is very granular.

selection-form

:lodgement-form a skos:Collection ;
    skos:definition "Earth science data categories for selection on lodgement form."@en ;
    skos:member 
    :geochemistry,
    :geochronology,
    :geology,
    :geophysics ;
    skos:prefLabel "Lodgement form selection"@en .

Also fixed the skos:inScheme errors

### CODE REVIEW

**Comment by DavidCrosswellGSQ** [comment]
Here is what was in the ModurlTech form:
image

**Comment by geoderekh** [comment]
@LukeHauck and @AdrianStead I'm looping you in here. Essentially the thinking here is that we want to make a collection of concepts for what to display in the lodgement portal so we can choose our level of granularity we want customers to tag their data at. David and I thought that 'get all concepts' made a very long select list and that a broader grouping would be better for the lodgement portal (still can tag

**Comment by Greenwoodmatthew** [comment]
First read i don't think this captures the requirement.  Some reports are based on a dataset, but many (EPM ann) are a report that will have none or many datasets included.  The tying of reports to ESDC is wrong, it should be tying datasets to ESDC, yes?

File: vocabularies/earth-science-data-category.ttl [hunk 1]
```diff
@@ -558,3 +557,11 @@
     skos:notation "040308" ;
     skos:prefLabel "Palaeontology"@en .
 
+:lodgement-form a skos:Collection ;
```
Comment (Greenwoodmatthew) [inline vocabularies/earth-science-data-category.ttl:560]: If we reduce the complexity of the ESDC for entry where will those vocabs be used?  How we will define data as a certain type? i.e. if a airborne survey is classified as geophysics on lodgement.  Where does it get defined as an airborne EM survey with a ESDC of electromagnetic methods in geophysics being more precise?   Similarly for seismic data.  How does it get tagged as seismic data from this?

**Comment by DavidCrosswellGSQ** [comment]
Reopening this pull request

**Comment by DavidCrosswellGSQ** [comment]
An alternative raised by Nick was to display concepts from the top two levels, i.e.:

This is a SPARQL query - so the vocabulary can remain as-is. No need for a collection.

Concept | Broader concept
-- | --
Atmospheric Sciences |  
Basin Analysis | geology
Biological Sciences |  
Demography |  
Ecology | biological-sciences
Electrical and Electromagnetic Methods in   Geophysics | geophysics
Engineering |  
Environmental Engineering | engineering
Environmental Sciences |  
Extract Geochemistry | geochemistry
Extraterrestrial Geology | geology
Geochemistry |  
Geochemistry Not Elsewhere Classified | geochemistry
Geochronology | geology
Geodynamics | geophysics
Geology |  
Geology Not Elsewhere Classified | geology
Geomechanics | geology
Geomechanics and Resources Geotechnical   Engineering | engineering
Geomorphology And Regolith And Landscape   Evolution | physical-geography
Geophysical Fluid Dynamics | geophysics
Geophysics |  
Geophysics Not Elsewhere Classified | geophysics
Geothermics and Radiometrics | geophysics
Glaciology | physical-geography
Gravity | geophysics
Hydrogeology | physical-geography
Inorganic Geochemistry | geochemistry
Isotope Geochemistry | geochemistry
Magnetics | geophysics
Marine Geoscience | geology
Mineralogy and Crystallography | geology
Natural Hazards | physical-geography
Organic Geochemistry | geochemistry
Palaeoclimatology | physical-geography
Palaeontology | geology
Petroleum and Coal Geology | geology
Petrology | geology
Petrophysics | geophysics
Physical Geography |  
Physical Geography And Environmental   Geoscience Not Elsewhere Classified | physical-geography
Quaternary Environments | physical-geography
Resource Geology | geology
Sedimentology | geology
Seismology and Seismic Exploration | geophysics
Spectral Imaging | geophysics
Stratigraphy | geology
Structural Geology | geology
Surface Processes | physical-geography
Surfacewater Hydrology | physical-geography
Tectonics | geology
Volcanology | geology

**Comment by DavidCrosswellGSQ** [comment]
Notwithstanding that we need to do more work on this vocab...

**Comment by Greenwoodmatthew** [comment]
I still don't know if i understand the point of this vocab.  Surely we need to tag the data objects as certain data types (seismic survey data, EM survey data, geochem data) at some point in lodgement so it can be picked up and inserted in geoprops.  To do this we need more than a top level geophysics tag.  If the ESDC tag (of lets say geophysics) is added to report, which object is the geophysics object?  And of the geophysics broader ESDC which of the narrower is used to tag the data object?    

The only point i see in this at all is that once data objects are tagged correctly (This is EM data, this is soil geochem data as those datasets are lodged) the report displays a tag saying it has broader geochem and geophsy data.  Like the current QDEX keywords functions.  But this is a search function, not a lodgement function in this way.  

Where am i missing this?

**Pull request merged**